api/src/Repository/DepartmentRepository.php

```diff
@@ -6,6 +6,7 @@
 use Atlas\Query\Select;
 use Atlas\Query\Insert;
 use Atlas\Query\Update;
+use Atlas\Query\Delete;
```

Comment:
This is not required anymore.

Reply:
All set. Removed `use Exception` as well since it was not used either.